app/controllers/supervisor_runs_controller.rb

```diff
@@ -151,11 +148,21 @@ def destroy
   private
```

Comment:
Tej metody nie można używać kilkukrotnie - już pojedyncze wywołanie powoduje, że wszystkie następujące po niej metody są prywatne.

" With no arguments, sets the default visibility for subsequently defined methods to private."

Zamiast tego na końcu pliku używa się private z listą sygnatur, np.

private :load_supervisor_run, :check_supervisor_owners